### PR TITLE
Mobile style improvements

### PR DESCRIPTION
**Pull request by zbettenbuk**

### CODE REVIEW

File: react/features/base/react/components/native/Header.js [hunk 1]
```diff
@@ -2,8 +2,13 @@
 
 import React, { Component, type Node } from 'react';
 import { Platform, SafeAreaView, StatusBar, View } from 'react-native';
+import { connect } from 'react-redux';
 
-import styles, { HEADER_PADDING, STATUSBAR_COLOR } from './styles';
+import { ColorSchemeRegistry } from '../../../color-scheme';
+
+// import { isDarkColor } from '../../../styles';
```
Comment (saghul) [inline react/features/base/react/components/native/Header.js]: Debug leftover?

File: react/features/base/react/components/native/headerstyles.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// @flex
+
+import { StyleSheet } from 'react-native';
+
+import { ColorSchemeRegistry, schemeColor } from '../../../color-scheme';
+import { BoxModel } from '../../../styles';
+
+const HEADER_FONT_SIZE = 18;
+const HEADER_HEIGHT = 48;
```
Comment (saghul) [inline react/features/base/react/components/native/headerstyles.js]: this used to be 56, why the change?

File: react/features/base/styles/functions.js [hunk 3]
```diff
@@ -136,6 +142,23 @@ export function getRGBAFormat(color: string, alpha: number): string {
...
+/**
+ * Decides if a color is light or dark based on the ITU-R BT.709 and W3C
+ * recommendations.
+ *
+ * NOTE: Please see https://www.w3.org/TR/WCAG20/#relativeluminancedef.
+ *
+ * @param {string} color - The color in rgb, rgba or hex format.
+ * @returns {boolean}
+ */
+export function isDarkColor(color: string): boolean {
```
Comment (saghul) [inline react/features/base/styles/functions.js]: ❤️

Comment (saghul) [reply react/features/base/styles/functions.js]: Looks like this is not used yet though?

File: react/features/base/react/components/native/Header.js (hunk 1 above)
Comment (zbettenbuk) [reply react/features/base/react/components/native/Header.js]: Nah, I just forgot to finish the implementation of the color luminance feature. Thanks!

File: react/features/base/styles/functions.js (hunk 3 above)
Comment (zbettenbuk) [reply react/features/base/styles/functions.js]: Yep, see above, thanks for pointing it out

File: react/features/base/react/components/native/headerstyles.js (hunk 2 above)
Comment (zbettenbuk) [reply react/features/base/react/components/native/headerstyles.js]: That's main the point of the commit "Make the header more compact".

The header was unnecessarily tall and that was very visible when someone wanted to apply a more native color scheme (for example iOS colors)

Comment (saghul) [reply react/features/base/react/components/native/headerstyles.js]: MD defines it as 56: https://material.io/design/components/app-bars-top.html#specs I know we are not implementing MD we are "getting inspired" by it. So are we ok with this change?

File: react/features/base/react/components/native/headerstyles.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+export const HEADER_PADDING = BoxModel.padding / 2;
+
+ColorSchemeRegistry.register('Header', {
+
+    disabledButtonText: {
```
Comment (saghul) [inline react/features/base/react/components/native/headerstyles.js]: add a comment please

File: react/features/base/react/components/native/headerstyles.js (hunk 2 above)
Comment (zbettenbuk) [reply react/features/base/react/components/native/headerstyles.js]: Well, I really love this new compact header, for me it looks good on Android, but it definitely looks way better on iOS. I'd say it's less platform specific now

**Comment by zbettenbuk** [comment]
The test passed, but github and jenkins got out of sync. Link to the green build: http://ci.jitsi.org/view/pr-testing/job/jitsi-meet-PR-test/5756/
